com.project_rk.launcher.yml

```diff
@@ -0,0 +1,229 @@
+        mkdir -p /app/lib/i386-linux-gnu
+        mkdir -p /app/lib/i386-linux-gnu/GL
+        mkdir -p /app/lib/i386-linux-gnu/dri/intel-vaapi-driver
+        mkdir -p /app/lib/debug/lib/i386-linux-gnu
+        mkdir -p /app/lib/debug/lib/i386-linux-gnu/GL
+        install -Dm644 -t /app/etc ld.so.conf
+        mkdir -p /app/lib{,32}/ffmpeg
+        mkdir -p /app/share/steam/compatibilitytools.d
+        mkdir -p /app/utils /app/share/vulkan
+        ln -srv /app/{utils/,}share/vulkan/explicit_layer.d
+        ln -srv /app/{utils/,}share/vulkan/implicit_layer.d
+        mkdir -p /app/links/lib
+        ln -srv /app/lib /app/links/lib/x86_64-linux-gnu
+        ln -srv /app/lib32 /app/links/lib/i386-linux-gnu
+    sources:
+      - type: inline
+        dest-filename: ld.so.conf
+        contents: |
+          # We just make any GL32 extension have higher priority
+          include /run/flatpak/ld.so.conf.d/app-*-org.freedesktop.Platform.GL32.*.conf
+          /app/lib32
+          /app/lib/i386-linux-gnu
+          /lib64
+
+  - name: umu
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 umu-run $FLATPAK_DEST/bin/umu-run
+      - install -Dm755 umu_run.py $FLATPAK_DEST/bin/umu-run.py
+    sources:
+      - type: archive
+        url: https://github.com/Open-Wine-Components/umu-launcher/releases/download/1.1.3/Zipapp.zip
+        sha256: bb26f7bce5291076c056dca9d0ded0bc0eb0b246d74e73b16e63808953b0f995
+
+  # Project RK Scripts
+  - name: project-rk
+    buildsystem: simple
+    build-commands:
+      - install -Dm 755 project-rk.sh ${FLATPAK_DEST}/bin/project-rk
+      - install -Dm 755 dgVoodooCpl.sh ${FLATPAK_DEST}/bin/dgVoodooCpl
+    sources:
+      - type: file
+        path: bin/project-rk.sh
+      - type: file
+        path: bin/dgVoodooCpl.sh # Added for dgvoodoo control panel config support
+
+  # metainfo, shortcut, and icons
+  - name: metadata
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 ${FLATPAK_ID}.desktop -t /app/share/applications # desktop shortcut
+      - install -Dm644 ${FLATPAK_ID}.dgVoodooCpl.desktop -t /app/share/applications # desktop shortcut for dgvoodoocpl
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml -t /app/share/metainfo # flathub metainfo
+      - install -Dm644 icon.svg /app/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      - install -Dm644 icon128.png /app/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - install -Dm644 icon48.png /app/share/icons/hicolor/48x48/apps/${FLATPAK_ID}.png
+      - install -Dm644 icon32.png /app/share/icons/hicolor/32x32/apps/${FLATPAK_ID}.png
+      - install -Dm644 icon24.png /app/share/icons/hicolor/24x24/apps/${FLATPAK_ID}.png
+      - install -Dm644 com.project_rk.launcher.dgVoodooCpl128.png /app/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.dgVoodooCpl.png
+      - install -Dm644 com.project_rk.launcher.dgVoodooCpl32.png /app/share/icons/hicolor/32x32/apps/${FLATPAK_ID}.dgVoodooCpl.png
```

Comment:
unless you need the smaller icons you should just ship svg + 128px png

Reply:
okay, I'll prune this part